infrastructure/sandbox.tfvars

```diff
@@ -1,47 +1,44 @@
+//module vars
+asp_name = "probate-sandbox"
+asp_rg = "probate-sandbox"
 env = "sandbox"
+external_host_name = "probate-caveats.sandbox.platform.hmcts.net"
+
+//app settings var
 deployment_env = "test"
 
-vault_section = "test"
+feature_toggles_api_url = "http://rpe-feature-toggle-api-demo.service.core-compute-demo.internal"
+
+idam_service_api = "http://betadevbccidams2slb.reform.hmcts.net"
```

Comment:
idam service host is chanfed

Reply:
changed it to idamservicehost from PA frontend

Reply:
idam_service_api changed to http://betadevbccidams2slb.reform.hmcts.net